backend/src/main/scala/sbt/internal/inc/bloop/internal/BloopAnalysisCallback.scala

```diff
@@ -354,7 +354,14 @@ final class BloopAnalysisCallback(
     }
   }
 
-  override def apiPhaseCompleted(): Unit = ()
+  override def apiPhaseCompleted(): Unit = {
+    // We collect all the generated classes during `API`. When this phase completes, we can pass to
```

Comment:
What about we replace all of this comment by this:
  
```
  /*
   * Inform the class file manager of the generated Scala classes as soon as
   * the Zinc API phase has run and collected them so that the class file 
   * invalidation registers these files before compiling Java files incrementally.
   */
```

Reply:
Sounds better :+1: